cmd/soroban-rpc/internal/config/options.go

```diff
@@ -211,18 +211,22 @@ func (cfg *Config) options() ConfigOptions {
 		},
 		{
 			Name: "event-retention-window",
-			Usage: fmt.Sprintf("configures the event retention window expressed in number of ledgers,"+
-				" the default value is %d which corresponds to about 24 hours of history", ledgerbucketwindow.DefaultEventLedgerRetentionWindow),
+			Usage: fmt.Sprintf(
```

Comment:
yeah I just didn't want to duplicate work in case all of us were going to make the same changes but all slightly differently - in fact we should probably target a separate branch for this work so that we can isolate changes like this one